### ISSUE TITLE
A11y_AzureToolsforVSCode_AzureFunction_FilterSubscriptions_ScreenReader: No name property announced for checkbox present beside the "Select Subscription" control.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/vscode-azurefunctions#2949]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:#A11yMAS;#A11yTCS;BM_AzureToolsforVSCode_Win32_Aug2021;#Azure Tools for VS
Code;#A11ySev2;#WCAG4.1.2;#Benchmark;#DesktopApp;#Win10;#NVDA;#JAWS;

### Environment Details:
Visual Studio Code
Azure Function
Operating System: Windows 10 Pro 21H1

### Steps to Reproduce:

1. Launch the Visual studio code.
2. TAB to on "Extension" button and search for "Azure Functions" and select it using ENTER key.
3. "Azure Function" screen should open.
4. TAB to "Azure" button and hit ENTER key.
5. Start NVDA and TAB to "Filter" button present in Subscriptions section.
6. TAB to checkbox present beside the "Select Subscription" control.
6. Observe the announcement by screen reader.

### Actual:
No name property announced by NVDA for checkbox present beside the "Select Subscription" control.
**This issue is also observed with JAWS.**

Expected:
The NVDA should announce the appropriate name property for checkbox present beside the "Select Subscription" control such as "Select All subscription" checkbox.

User Impact:
Screen reader users won't be able to understand the purpose of the control.

**Comment by nturinski** [comment]
This seems to be a duplicate of this issue https://github.com/microsoft/vscode-azurefunctions/issues/2947.  As mentioned in the linked issue, this is an external bug that we cannot control.

**Comment by 12-shweta** [comment]
@nturinski 
As you mentioned that this bug is duplicate of the [https://github.com/microsoft/vscode-azurefunctions/issues/2947](url) but here the control seems to be different. Please find the screenshot for reference
![image](https://user-images.githubusercontent.com/54656883/144000191-84e9763a-f254-46e4-a998-27a3f0cacb26.png)

**Comment by nturinski** [comment]
I see, however this isn't something that the extension can fix.  Please file an issue on https://github.com/microsoft/vscode/issues if you believe that the control being different is enough reason to track as a separate issue.

**Comment by 12-shweta** [comment]
@nturinski 
Please find the external bug logged for this issue : https://github.com/microsoft/vscode/issues/143855

**Comment by 12-shweta** [comment]
@nturinski 
We logged external bug but got the below comment:
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!

Could you please check once

**Comment by 12-shweta** [comment]
@nturinski
external bug got closed stating the below comment:
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!

Could you please check once

**Comment by nturinski** [comment]
@12-shweta We are currently investigating this.  I'll let you know when have progressed this issue.

**Comment by MicroFish91** [comment]
Filed an upstream issue that should cover this one, tracking it here: https://github.com/microsoft/vscode/issues/149884

**Comment by 12-shweta** [comment]
@MicroFish91 We tried to verify the bug in the Visual studio code but we are still able to reproduce the issue.
![image](https://user-images.githubusercontent.com/54656883/169801572-fb45c117-be61-49d2-8c2a-bc3917348df4.png)

**Comment by MicroFish91** [comment]
I believe the VS Code team made the changes under version 1.68

**Comment by jinglouMSFT** [comment]
@12-shweta Could you please regress the fix using the latest VS Code insider build together with the latest release of the Functions extension? Resolve and close this bug if it's fixed in the VS Code side.  Thanks.

**Comment by Rama8345** [comment]
**Github Tags:**#Closed;

**Comment by InduPriya1805** [comment]
GitHubTags:#BM_AzureToolsforVSCode_Win32_Aug2021;
